libraries/stdlib/src/kotlin/io/Console.kt

```diff
@@ -132,10 +132,9 @@ public inline fun println() {
     System.out.println()
 }
 
-// Since System.in can change its value on the course of program running,
-// we should always delegate to current value and cannot just pass it to InputStreamReader constructor.
+// Since System.in can change its value on the course of program running, we should always delegate to current value.
 // We could use "by" implementation, but we can only use "by" with interfaces and InputStream is abstract class.
-private val stdin: BufferedReader by lazy { BufferedReader(InputStreamReader(object : InputStream() {
+private val stdin: InputStream by lazy { object : InputStream() {
```

Comment:
Since we're not wrapping it in a reader anymore, we could call `read` methods directly on `System.in` and remove this val entirely.

Reply:
I personally like this particular change however isn't it a breaking change?

Reply:
It won't break anything because readLine is the only place where it is used. But leaving it can be useful in future, considering [KT-11207](https://youtrack.jetbrains.com/issue/KT-11207). Anyway I should try to simplify this declaration.